### ISSUE TITLE
Multi uploader asset type problem

### ISSUE THREAD

**Issue by ismailakincom** [didiatworkz/screenly-ose-monitor#111]
What's your question?
When we are trying to use multi uploader with a website (ex. https://test-videos.co.uk/vids/bigbuckbunny/mp4/h264/1080/Big_Buck_Bunny_1080_10s_1MB.mp4), asset type is appearing as web pages. When we are using anthias single loader it is appearing as video. Could you help us with that? We want it to appear like Video on Asset Type when we are using multi uploader.

**Comment by ismailakincom** [comment]
https://developer.screenly.io/api/#tocS_AssetUpdate  


Property | Value
-- | --
type | image
type | video
type | appweb
type | web
type | audio
type | edge-app

**Comment by kadirkilicnet** [comment]
Yes, definetly. There should be Asset Type selection function in multi uploader.
